src/ui/widgets/tui_dirlist_detailed.rs

```diff
@@ -103,10 +134,19 @@ fn print_entry(
         drawing_width,
     );
 
-    let right_width = right_label.width();
-    buf.set_stringn(x, y, left_label, drawing_width, style);
+    // Drawing index (If not empty)
```

Comment:
I think theres some weird bugs going on here with calculating margins, spacing, etc.
But I'm guessing you're still ironing it out :+1: